Some updates (and additions)

jaju opened the pull request:
Hi @DaveWM 

Thanks for the wonderful effort! I was playing around and updated dependencies, plus a few more things gleaned from @taylorSando's repo and other places to get rid of warnings I'd see. But honestly, some of the changes (like double import of React related fixes) were more of trials-and-errors.

Updated the tests too with more components - they pass with phantomjs v2.1.1.

Do feel free to accept this pull request (or not!)

Cheers!

jaju committed:
'Hopeless shooting in the dark' fixes for warnings/errors

When they started to show up with the various (latest) versions I
switched to for consistency/compatibility (as I use them elsewhere.)

jaju committed:
Comment explaining the empty cljsjs.react ns - taylorSando/om-material-ui

jaju committed:
Removed some blindly copied components - which apparently are not exported.

jaju committed:
lein doo phantom passes - phantomjs version = 2.1.1

jaju committed:
Updated README and added a note saying that this fork is not (yet) published to clojars. Should send a pull request to @DaveWM next.

DaveWM commented:
@jaju Thanks very much for the PR :). I'll take a look through it at some point this week and let you know my thoughts.

jaju committed:
Fix for exporting the 'List' component

No solution yet for 'Stepper.'
A few version updates too for the material-ui builder.

jaju committed:
Oops.

DaveWM commented:
@jaju I've merged your PR, just a few points:
- I've downgraded the reagent and material-ui versions to stable versions. I'd prefer to get everything working and more or less complete with stable dependencies, then create another branch using alpha versions of the dependencies. 
- I've decided to remove react-addons completely. Since they're not needed for material ui, and they're in separate npm packages, we shouldn't really include them in the build.
- I've also bumped the project version to 0.2.0 due to the above breaking change

Again, thanks for the PR :+1:

jaju commented:
Thanks, @DaveWM! Makes sense.
Maybe, you can also update the README to reflect the 0.2.0-SNAPSHOT as the new version line?
